python/ray/ml/constants.py

```diff
@@ -0,0 +1,5 @@
+# Key to denote the preprocessor in the checkpoint dict.
+PREPROCESSOR_KEY = "_preprocessor"
```

Comment:
@krfricke FYI, I now remember why we had "metadata" as a separate field--- so that preprocessors can be loaded without needing to deserialize the entire model.

I think this isn't that important for alpha though, just a deficiency to keep in mind.